Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -59,6 +61,10 @@
             Thread.currentThread().setContextClassLoader(AuthMethodManager.class.getClassLoader());
             HttpClientProviders.createInstance();
             com.microsoft.azure.toolkit.lib.Azure.az(AzureAccount.class);
+            if (CommonSettings.getEnvironment() != null) {
```

Comment:
`CommonSettings.env` was set up in[ `AzureActionsListener`](https://github.com/microsoft/azure-tools-for-java/blob/develop/PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzureActionsListener.java#L101), please verify the init sequence for `AuthMethodManager` and `AzureActionsListener`

Reply:
checked, first init CommonSettings and then init azure cloud